jhub_apps/service/initialize_startup_apps.py

```diff
@@ -0,0 +1,128 @@
+import asyncio
+from itertools import groupby
+import signal
+import time
+
+import httpx
+import structlog
+
+from jhub_apps.configuration import FASTAPI_SERVICE_NAME
+from jhub_apps.hub_client.hub_client import HubClient
+from jhub_apps.service.models import StartupApp
+from jhub_apps.service.utils import get_jupyterhub_config
+
+logger = structlog.get_logger(__name__)
```

Comment:
You would need to call [setup_logging](https://github.com/nebari-dev/jhub-apps/blob/60d3a7b0fb8eaced6949e4ef19db33b675ebfb3b/jhub_apps/service/logging_utils.py#L6) in this service as well to get the same logging config as the fast api service.